Pack-Smart/pack-smart-fe
# Testing/fetches

## What's this PR do?
- mocks `fetch` on all api calls and tests what it was called with at each call.
- Creates a new directory for apiCalls that houses apiCalls.js and apiCalls.test.js
- Adds more sample data to make the tests

## Where should the reviewer start?
- apiCalls.test.js
- sampleData.js

## How should this be manually tested?
- Run `npm test` to confirm all the tests are passing.
- Confirm the api is not actually being called with an actual fetch.

## Any background context you want to provide?
- This took a long time to figure our.

## What are the relevant tickets?
- closes #60 

## Did you update the README?
- no
